gce/startup-script.sh

```diff
@@ -0,0 +1,70 @@
+#! /bin/bash
+#	Copyright 2019, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START startup]
+set -v
+
+# Talk to the metadata server to get the project id
+PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
+REPOSITORY="[YOUR-REPOSITORY]"
+
+# Install logging monitor. The monitor will automatically pick up logs sent to
+# syslog.
+# [START logging]
```

Comment:
I didn't know we can samples interspersed like this, this would be a `logging` sample inside a `startup` snippet?